### PR TITLE
change : formula typo updated in pizza pi

### PR DESCRIPTION
**Pull request by fejan92**
Updated the typo as suggested in the below issue 

https://github.com/exercism/php/issues/616

### CODE REVIEW

File: exercises/concept/pizza-pi/.docs/instructions.md [hunk 1]
```diff
@@ -50,15 +51,16 @@ $pizza_pi->calculateSauceRequirement(8, 250);
 ## 3. Some Cheese, Please
 
 Cheese comes in perfect cubes and is sold by size.
-Lilly calculated an equation to determine how many pizzas can  of some diameter (d) made from a cheese cube of side-length (l):
+Lilly calculated an equation to determine how many pizzas can of some diameter (d) made from a cheese cube of side-length (l):
```
Comment (mk-mxp) [inline exercises/concept/pizza-pi/.docs/instructions.md:54]: This is one the 2 changes to the text, we want to have:

```suggestion
Lilly decided to use the following formula to determine how many pizzas of some diameter (d) can be made from a cheese cube of side-length (l):
```

File: exercises/concept/pizza-pi/.docs/instructions.md [hunk 2]
```diff
@@ -50,15 +51,16 @@ $pizza_pi->calculateSauceRequirement(8, 250);
...
 Create a function that:
-* Takes a side-length dimension of a cheese cube
-* Takes the desired thickness of the cheese layer
-* Takes the diameter of the pizza
-* And returns the number of pizzas that can be made while rounding down.
+
+- Takes a side-length dimension of a cheese cube
+- Takes the desired thickness of the cheese layer
+- Takes the diameter of the pizza
+- And returns the number of pizzas that can be made while rounding down.
```
Comment (mk-mxp) [inline exercises/concept/pizza-pi/.docs/instructions.md:63]: And this is the second piece of text we would like to change, so it is more clear that the formular shown above must be implemented (and not the area of the pizza).

```suggestion
- And uses Lilly's formula to return the number of pizzas that can be made while rounding down.
```

**Review by mk-mxp — CHANGES_REQUESTED** [review]
I hope I could clarify, what changes are needed. And thanks very much for adjusting the Markdown formatting!

**Comment by fejan92** [comment]
I just committed the changes you requested @mk-mxp.

Please review it. Thanks!

**Review by mk-mxp — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**